diff --git a/Makefile.istep0 b/Makefile.istep0
new file mode 100644
index 0000000..47fc3a5
--- /dev/null
+++ b/Makefile.istep0
@@ -0,0 +1,2 @@
+# istep0
+include ekb/chips/p10/procedures/hwp/perv/p10_start_cbs.mk
diff --git a/Makefile.generate b/Makefile.generate
index c6ef9e8..d4c9f28 100644
--- a/Makefile.generate
+++ b/Makefile.generate
@@ -112,7 +112,7 @@ endef
 
 all: definitions
 
-include Makefile.istep
+include Makefile.istep0
 
 SLIST += $(SRC_EXTRA)
 
